tool/gocross/autoflags.go

```diff
@@ -102,6 +102,10 @@ func autoflagsForTest(argv []string, env *Environment, goroot, nativeGOOS, nativ
 		cgo = nativeGOOS == "darwin"
 		tags = append(tags, "omitidna", "omitpemdecrypt")
 		if env.IsSet("XCODE_VERSION_ACTUAL") {
+			// If we're building via Xcode, we must be making the extension
+			// version (as opposed to tailscaled on Mac).
+			tags = append(tags, "ts_macext")
```

Comment:
Does ts_macext cover the App Store build as well?

Reply:
As far as I can tell it was specified for all the Xcode builds (iOS, macOS, macsys): https://github.com/tailscale/corp/blob/72401f9d3bd6eecfde5f5a2cb70a5469fb012bf0/xcode/ipn-go-bridge/default.a.do#L43-L51.